starter-code/views/passport/login.ejs

```diff
@@ -0,0 +1,13 @@
+<form action="/login" method="POST">
+    <div>
+        <label for="username">Username:</label>
```

Comment:
If you use `for` in your `label` tag you need an element with the corresponding `id`.